docs/reference/team-processes/triage-process.md

```diff
@@ -15,7 +15,7 @@ Triage owners rotate throughout the team, and at the time of writing the triage
 
 ## I am the Triage Owner: How should I plan my Sprint?
 
-When taking on work for the sprint, consider taking on around 50% less than usual in order to allocate a portion of each day to triage ownership duties. If you're in the middle of high priority feature work, fill your availability with that work. Otherwise, favor taking on a high priority maintenance item(s).
+When taking on work for the sprint, if you’re on the FxA team, consider taking on around 50% less than usual in order to allocate a portion of each day to triage ownership duties. If you’re on the SubPlat team, consider taking on around 20% less than usual to assist with dependency updates or high priority bugs that come up, explained below. If you’re in the middle of high priority feature work, fill your availability with that work. Otherwise, favor taking on a high priority maintenance item(s).
```

Comment:
@annasob this is that update we talked about. This is still in a draft so feel free to check back tomorrow but I wanted to call this bit out.